### PR TITLE
fix: akka-stream-testkit version

### PR DESCRIPTION
**Pull request by vitaliis**
The old one (19) results in:
sbt.librarymanagement.ResolveException: unresolved dependency: com.typesafe.akka#akka-stream-testkit_2.13;2.5.19: not found

## References

References #xxxx

### CODE REVIEW

File: docs/src/main/paradox/routing-dsl/testkit.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ route logic easy and convenient. This "route test DSL" is made available with th
 To use Akka HTTP TestKit, add the module to your project:
 
 @@dependency [sbt,Gradle,Maven] {
-  group="com.typesafe.akka" artifact="akka-stream-testkit_$scala.binary.version$" version="2.5.19"
+  group="com.typesafe.akka" artifact="akka-stream-testkit_$scala.binary.version$" version="2.5.23"
```
Comment (jrudolph) [inline docs/src/main/paradox/routing-dsl/testkit.md]:
```suggestion
  group="com.typesafe.akka" artifact="akka-stream-testkit_$scala.binary.version$" version="$akka.version$"
```

**Comment by jrudolph** [comment]
OK TO TEST

File: docs/src/main/paradox/routing-dsl/testkit.md (hunk 1 above)
Comment (raboof) [reply docs/src/main/paradox/routing-dsl/testkit.md]: Hmm, the version we recommend in the docs might still be higher than the version we really require though... perhaps we should introduce yet another variable for this?

Comment (jrudolph) [reply docs/src/main/paradox/routing-dsl/testkit.md]: Wouldn't we always recommend the latest? Right now, we seem to use `akka.version` throughout, so maybe let's keep it consistent for now?

I guess we can think about always recommending the latest version of akka?

Comment (raboof) [reply docs/src/main/paradox/routing-dsl/testkit.md]: Yeah, not necessary for this PR indeed ;)

**Pull request merged**